Jenkinsfile

```diff
@@ -662,7 +663,9 @@ def withBeatsEnv(boolean archive, Closure body) {
       sh(label: "Install docker-compose ${DOCKER_COMPOSE_VERSION}", script: ".ci/scripts/install-docker-compose.sh")
       sh(label: "Install Mage", script: "make mage")
       try {
-        body()
+        if(!params.dry_run){
```

Comment:
Can the `dry_run` move to the beginning of the methods: `withBeatsEnvWin ` and `withBeatsEnv`? Or is it required to run the sh/bat steps?

Reply:
in that case we do not test the installer.